packages/aragon-wrapper/src/index.js

```diff
@@ -935,9 +935,23 @@ export default class Aragon {
    *
    * @return {Promise<void>}
    */
+  
   clearLocalIdentities () {
     return this.identityProviderRegistrar.get('local').clear()
   }
+  
+  
+  /**
+   * Clear selected local identities
+   *
+   * @return {Promise<Object>}
+   */
+  
+  removeSelectedLocalIdentities (selectedAddresses) {
```

Comment:
This will probably need to be `async` no?

Reply:
Yes. The function needs to be async.

I'd also rename it to just `removeLocalIdentities`